subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -5715,7 +5715,7 @@ static void mayfly_sched_win_offset_select(void *params)
 		struct pdu_data *pdu_ctrl_tx;
 
 		/* procedure request acked */
-		conn->llcp_ack = conn->llcp_req;
+		atomic_cas(&conn->llcp_ack, conn->llcp_ack, conn->llcp_req);
```

Comment:
Could you maybe elaborate why the return value of atomic_cas() is not checked here? As far as I can tell, the update would simply be lost in case of a race.

Reply:
Yes, should be a CAS loop to make sense. (But here I'm not sure if we need cas loop at all; @cvinayak ?)